server/chat-commands/room-settings.ts

```diff
@@ -205,13 +157,12 @@ class RoomSettings {
 		output += `<strong>Caps filter:</strong> <br />${this.capitals()}<br />`;
 		output += `<strong>Emoji filter:</strong> <br />${this.emojis()}<br />`;
 		output += `<strong>Slowchat:</strong> <br />${this.slowchat()}<br />`;
```

Comment:
Perhaps all settings should use the same syntax? Maybe official settings should also be registered in `Chat.roomSettings`.